### PR TITLE
chuangmi.plug.v3: Fixed power state status for updated firmware

### PR DESCRIPTION
**Pull request by ad**
from [this comment](https://github.com/home-assistant/core/issues/31653#issuecomment-611667270)

### CODE REVIEW

**Comment by rytilahti** [comment]
I don't think this this really fixes the issue, considering that the first check will always return True for non-empty string values too (i.e., `off` would be matched, too), right?

File: miio/chuangmi_plug.py [hunk 1]
```diff
@@ -46,7 +46,7 @@ def __init__(self, data: Dict[str, Any]) -> None:
     def power(self) -> bool:
         """Current power state."""
         if "on" in self.data:
-            return self.data["on"]
+            return self.data["on"] or self.data["on"] == "on"
```
Comment (rytilahti) [inline miio/chuangmi_plug.py:49]:
```suggestion
            return self.data["on"] is True or self.data["on"] == "on"
```

**Comment by matteos1** [comment]
is it possible to correct this error in home assistant?

**Comment by rytilahti** [comment]
It will correct itself when we make a new release, and homeassistant updates to that.

**Comment by matteos1** [comment]
in 0.109?

**Comment by rytilahti** [comment]
No, 0.109 beta is already out so it will likely land in 0.110.

**Comment by matteos1** [comment]
is not possisible in future release of 0.109? thank you

> No, 0.109 beta is already out so it will likely land in 0.110.

**Comment by rytilahti** [comment]
I wanted to get some of the currently open PRs merged first, but maybe I'll cut a new release over the weekend and see if we can get it into some .x homeassistant release!

**Comment by matteos1** [comment]
thank you

**Comment by matteos1** [comment]
> I wanted to get some of the currently open PRs merged first, but maybe I'll cut a new release over the weekend and see if we can get it into some .x homeassistant release!

same problem on 0.110 the switch is already on :(

**Comment by matteos1** [comment]
> I wanted to get some of the currently open PRs merged first, but maybe I'll cut a new release over the weekend and see if we can get it into some .x homeassistant release!

have  you update chuangmi_plug.py?

**Comment by acelauwc** [comment]
As of today, its still not in the home assistant release 0.114.4 

For those who cannot wait, you can manually edit chuangmi_plug.py (line 49) in your home assistant miiio directory to reflect the change.

**Pull request merged**